package.json

```diff
@@ -18,7 +18,9 @@
     "test:watch": "jest --coverage --watch",
     "test:all": "set CI=true && yarn test && yarn lint",
     "cy:run": "cypress run",
-    "cy:open": "cypress open"
+    "cy:open": "cypress open",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
should use build:storybook instead of build-storybook by following other commands